doc/src/manual/integers-and-floating-point-numbers.md

```diff
@@ -286,7 +286,7 @@ Float64
 Half-precision floating-point numbers are also supported (`Float16`), but only as a storage format.
 In calculations they'll be converted to `Float32`:
```

Comment:
It is still partially true, since the underlying implementation of `Float16` is using `Float32`. I would change this too

>  Half-precision floating-point numbers are also supported (`Float16`), but they are implemented in software and use `Float32` for calculations.

The change that made 79a2e01 necessary was that operations with `Float16` now return `Float16` allowing us in the future to use hardware implementations.